media_numeros_array.rb

```diff
@@ -0,0 +1,17 @@
+rand_array = []
+sum = 0
+media = 0
+
+10.times do |index|
+  rand_array.push(rand(0..100))
+end
+
+puts "#{rand_array}"
+
+rand_array.each_with_index do |value, index|
+  sum = rand_array[index] + sum
+end
+puts "A soma dos valores do array é: #{sum}"
```

Comment:
```suggestion
rand_array.each do |value|
  sum += value
end

media = sum / rand_array.length
puts "A soma dos valores do array é: #{media}"
```

Eu não sei se seria para somar ou seria a média, por que no nome do arquivo diz media numeros, mas você fez a soma, certo?

Lhe deixo uma maneira melhor de se fazer e com a média já.

Reply:
Não sei o pq, mas o código desse exercício tá no arquivo errado. Devo ter duplicado o arquivo e na hora de jogar pro git enviei o arquivo errado...sei lá o que eu fiz kkkk. Mas valeu!